public/docs/ts/latest/guide/user-input.jade

```diff
@@ -37,27 +40,27 @@ include ../_util-fns
 .l-main-section
 :marked
   ## Get user input from the $event object
-  We can bind to all kinds of events. Let's bind to the keyup event of an input box and replay
+  You can bind to all kinds of events. In this case, you'll bind to the keyup event of an input box and replay
   what the user types back onto the screen.
 
-  This time we'll (1) listen to an event and (2) grab the user's input.
+  This time, you'll listen to an event and grab the user's input.
```

Comment:
The only problem with using imperative here is that the reader might think that they're being ordered to do something (which they don't know yet how to do yet). This is the one case where future tense (and possibly even "let's") might be the way to go.